RedditSharp/Reddit.cs

```diff
@@ -317,6 +319,17 @@ public Comment GetComment(string subreddit, string name, string linkName)
             }
         }
 
+        public Listing<T> SearchByUrl<T>(string url) where T : Thing
+        {
+            var urlSearchQuery = string.Format(UrlSearchPattern, url);
+            return Search<T>(urlSearchQuery);
+        }
+
+        public Listing<T> Search<T>(string query) where T : Thing
```

Comment:
Can you give me a specific example for this statement? Eg: the keyword is "beautiful". Thanks